Lib/typing.py

```diff
@@ -1,20 +1,21 @@
 """
-The typing module: Support for gradual typing as defined by PEP 484.
+The typing module: Support for gradual typing as defined by PEP 484 and subsequent PEPs.
 
 At large scale, the structure of the module is following:
 * Imports and exports, all public names should be explicitly added to __all__.
 * Internal helper functions: these should never be used in code outside this module.
 * _SpecialForm and its instances (special forms):
-  Any, NoReturn, Never, ClassVar, Union, Optional, Concatenate, Unpack
-* Classes whose instances can be type arguments in addition to types:
-  ForwardRef, TypeVar and ParamSpec
+  NoReturn, Never, ClassVar, Self, Concatenate, Unpack, and others.
+* Classes whose instances can be type arguments:
+  TypeVar, ParamSpec, TypeVarTuple.
 * The core of internal generics API: _GenericAlias and _VariadicGenericAlias, the latter is
```

Comment:
`_VariadicGenericAlias` doesn't exist (and even if it did, we shouldn't mention all this private stuff).